infoschema/tables.go

```diff
@@ -995,6 +1010,7 @@ func dataForTables(ctx sessionctx.Context, schemas []*model.DBInfo) ([][]types.D
 					nil,           // CHECKSUM
 					nil,           // CREATE_OPTIONS
 					"VIEW",        // TABLE_COMMENT
+					nil,           // TIDB_TABLE_ID
```

Comment:
View also have table ID. But the column name is `TIDB_TABLE_ID`, return nil here is ok for me.